runtime/Cargo.toml

```diff
@@ -46,6 +47,7 @@ solana-sdk = { path = "../sdk", version = "1.5.0" }
 solana-stake-program = { path = "../programs/stake", version = "1.5.0" }
 solana-vote-program = { path = "../programs/vote", version = "1.5.0" }
 solana-secp256k1-program = { path = "../programs/secp256k1", version = "1.5.0" }
+spl-token-v2-0 = { package = "spl-token", version = "=2.0.8" }
```

Comment:
Would prefer to avoid this dependency if possible. Let's move the new runtime functionality into a module in `solana-transaction-status`, and BankingStage and BlockstoreProcessor can call it from there. Both solana-core and solana-ledger both depend on transaction-status already